### DISCUSSION TITLE
How do I run a specific action?

### DISCUSSION THREAD

**Discussion by huseyin-hamilton-cko** [nektos/act#1288] [Q&A]
I have never used act before so I could be using it incorrectly. 

I just want to test the run of one action in a branch which isn't merged to "main" yet. 

On my branch locally, I run something like:
`act -v --defaultbranch branch-name -j testing-build.yml`

The last line it outputs is:
```
DEBU[0000] Planning job: testing-build.yml
```
And then does nothing. Any help/ideas about what I should be doing?

**Comment by catthehacker** [comment]
you didn't specify correct event that triggers the workflow
